app/models/event.rb

```diff
@@ -0,0 +1,15 @@
+class Event < ApplicationRecord
+  validates :title, :place_title, :place_cord, :description, :start_at, :end_at, presence: true
+  validates :title, length: { in: 6..20 }
+  validates :place_title, length: { maximum: 30 }
+  validates :description, length: { minimum: 30 }
+  validate :end_after_start
```

Comment:
Confusing custom validation name.
Let's meditate on this.

Reply:
I suppose we gonna have the same validation for the Event::Time model, so maybe it worth to create a validation class for this case?

https://edgeguides.rubyonrails.org/active_record_validations.html#performing-custom-validations